dashboard/app/models/user.rb

```diff
@@ -2453,7 +2454,7 @@ def reset_section_attempts?
     !section_attempts_last_reset || num_section_attempts == 0 || (DateTime.now - DateTime.parse(section_attempts_last_reset)).to_i > 0
   end
 
-  def display_captcha?
+  def display_join_section_captcha?
```

Comment:
Renamed this to disambiguate from this new feature. Part of some unfinished work from 2+ years ago from someone who no longer works here to prevent spam joining of sections, further documented here:

https://codedotorg.atlassian.net/browse/LP-1717